pyproject.toml

```diff
@@ -5,24 +5,26 @@ description = ""
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.10"
-pypfb = ">=0.5.21"
-fastavro = "<=1.4.1"
-requests = "^2.25.1"
-psycopg2-binary = ">=2.8.4"
+
+python = ">=3.9.0,<3.10"
+
+project-request-scripts = {git = "https://github.com/chicagopcdc/project_request_scripts.git", rev = "add-poetry-project"}
+fastavro = "~1.8.2"
+requests = "^2.32.3"
 pyspark = "^3.0.1"
+gen3datamodel = ">=3.2.1"
+dictionaryutils = "^3.4.1"
+boto3 = "^1.38.3"
+sqlalchemy = ">=1.3,<2.0"
+pypfb = "^0.5.30"
+psycopg2-binary = ">=2.8.4"
```

Comment:
Unless strictly necessary revert the versions to the one we inherited from CTDS. https://github.com/uc-cdis/pelican/blob/master/pyproject.toml

Reply:
lined everything up